internal/policies/mount/mount.go

```diff
@@ -114,6 +135,17 @@ func (m *Manager) ApplyPolicy(ctx context.Context, objectName string, isComputer
 				log.Debugf(ctx, "Key %q is currently not supported by the mount manager", entry.Key)
 			}
 		}
+	} else {
+		for _, entry := range entries {
+			switch entry.Key {
+			case "system-mounts":
+				if e := m.applySystemMountsPolicy(ctx, objectName, entry); e != nil {
+					err = e
+				}
+			default:
+				log.Debugf(ctx, "Key %q is currently not supported by the mount manager", entry.Key)
```

Comment:
Honestly, I will refactor this section of the ApplyPolicy because I really hate it (I'll do it while Didier is not looking).

Reply:
I am looking :) (and you told in a previous PR that you were happy how we structured it in the user mount so that the number of changes here were minimal :p)

Reply:
(TBH here, entry.Key had "system" or "user" prefix in it)